includes/Captcha/Providers/CloudflareTurnstileProvider.php

```
@@ -0,0 +1,157 @@
+        return $html;
+    }
+
+    /**
+     * Validate a Turnstile response token via Cloudflare verification API.
+     *
+     * @param string $context Context key (not used by Turnstile but kept for interface parity).
+     * @param string $token   Token returned by Turnstile widget.
+     *
+     * @return bool True if verification is successful, false otherwise.
+     */
+	public function validate( string $context, string $token ): bool {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$context'. (undefined)

(UnusedFormalParameter)